### DISCUSSION TITLE
Nomination of @ Skinah

### DISCUSSION THREAD

**Discussion by hmerk** [openhab/openhab-addons#14708] [Add-on Maintainers Discussions]
Dear @openhab/add-ons-maintainers,
I would like to nominate @Skinah to become a openhab-addons maintainer.
Metthew has been very active reviewing open PR‘s and his comments show me a good Java and openHAB code knowledge.

Please vote with 👍 or 👎 here on this post. Thanks!

**Comment by wborn** [comment]
I don't think you are a maintainer @hmerk. :wink:

https://www.openhab.org/docs/developer/governance.html#nominations

> Only the component's maintainers may nominate a new maintainer or vote in a maintainer election.

**Comment by hmerk** [comment]
Ok, in that case, ignore my suggestion. Was just trying to help…
